docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -187,7 +187,9 @@ The following table explains the parameter values that you need to provide in st
 |Instance root volume size |15 GB or more is recommended. |
 |Key Name for SSH |Your EC2 key pair for SSH access to the deployed EC2 instances. |
 |Trusted Network CIDR |The CIDR (Classless Inter-Domain Routing) value determines the IP address range that can access your operator service.<br/>To limit access to the UID2 Operators so that they can only be accessed through an internal network or a load balancer, specify an internal IP range as the CIDR value. |
-
+|VPC |The existing VPC id. |
+|VpcSubnet1 |The existing VPC AZ1 SubnetId. |
```

Comment:
Is SubnetId a specific code element?
If yes, use the ticks for code style (`SubnetId`).
If no, say Subnet ID as we did earlier (line 122). That would definitely be my preference. Suggest:
|VpcSubnet1 |The existing VPC AZ1 Subnet ID. |